testsuite/features/step_definitions/salt_steps.rb

```diff
@@ -271,15 +271,19 @@
   # Complicated code because the checkbox is not a <input type=checkbox> but an <i>
   xpath_query = "//a[@id = '#{formula}']/i[@class = 'fa fa-lg fa-square-o']" if action == 'check'
   xpath_query = "//a[@id = '#{formula}']/i[@class = 'fa fa-lg fa-check-square-o']" if action == 'uncheck'
-  puts "Selected formulas [beginning]:\n#{find('#chooseFormulas')['innerHTML']}"
+  # WORKAROUND
+  # DOM refreshes content of chooseFormulas element by accessing it. Then conditions are evaluated properly.
+  find('#chooseFormulas')['innerHTML']
```

Comment:
We can get rid of that one.

Reply:
No. We must keep the first one and remove the others.

Reply:
Yep, this is important. Well if there is one single important line in this PR, it is this one :-D